### PR TITLE
update the FAQ

### PR DESCRIPTION
**Pull request by siunies**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/turning-on-and-using-call-quality-dashboard.md [hunk 1]
```diff
@@ -484,9 +484,17 @@ To compute call count and user count measures, a distinct countif operation is p
 
 If you see data differences between CQD v2 and v3, make sure that data comparison or validation is done on an 'apples-to-apples'  and narrow level, not an aggregated level. For example, if you filter both reports for MSIT 'Building 30' WiFi Teams Desktop client data, the Percentage of Poor Quality should be the same between v2 and v3.
 
-CQD v2 and CQD v3 have different total counts since CQD v3 has new scenarios not present in CQD v2. Summary Total or Aggregated all-up numbers with no filters are expected to be different.  
+CQDv2 classification for CallSetup failure is inly considered for "Audio" modality only, in CQDv3 this classification happens for every modality (Audio, Video and Appsharing) and is represented in the respective modality stream.
```
Comment (JohanFreelancer9) [inline Teams/turning-on-and-using-call-quality-dashboard.md:487]:
```suggestion
CQDv2 classification for CallSetup failure is considered for "Audio" modality only; in CQDv3 this classification happens for every modality (Audio, Video, and Appsharing) and is represented in the respective modality stream. 
```

File: Teams/turning-on-and-using-call-quality-dashboard.md [hunk 2]
```diff
@@ -484,9 +484,17 @@ To compute call count and user count measures, a distinct countif operation is p
 
 If you see data differences between CQD v2 and v3, make sure that data comparison or validation is done on an 'apples-to-apples'  and narrow level, not an aggregated level. For example, if you filter both reports for MSIT 'Building 30' WiFi Teams Desktop client data, the Percentage of Poor Quality should be the same between v2 and v3.
 
-CQD v2 and CQD v3 have different total counts since CQD v3 has new scenarios not present in CQD v2. Summary Total or Aggregated all-up numbers with no filters are expected to be different.  
+CQDv2 classification for CallSetup failure is inly considered for "Audio" modality only, in CQDv3 this classification happens for every modality (Audio, Video and Appsharing) and is represented in the respective modality stream. 
 
-If the usage scenario includes Skype for Business Server 2019 calls, CQD v3 data includes Skype Bot calls (auto attendant, CVI, Virtual Desktop Interface), Live Events, and PSTN calls. CQD v2 does not use this data. (CQD v3 requires Skype for Business Server 2019 with cloud data connector configured.)
+For Teams, CQDv2 applies the same user feedback to all modalities CQDv3 applies feedback base on modality for Teams.
```
Comment (JohanFreelancer9) [inline Teams/turning-on-and-using-call-quality-dashboard.md:489]:
```suggestion
For Teams, CQDv2 applies the same user feedback to all modalities. CQDv3 applies feedback base on the modality for Teams.
```

File: Teams/turning-on-and-using-call-quality-dashboard.md [hunk 3]
```diff
@@ -484,9 +484,17 @@ To compute call count and user count measures, a distinct countif operation is p
...
+
+CQD V3 includes
```
Comment (JohanFreelancer9) [inline Teams/turning-on-and-using-call-quality-dashboard.md:491]:
```suggestion
CQD V3 includes 
```
```suggestion
CQD v3 includes 
```

Comment (JohanFreelancer9) [inline Teams/turning-on-and-using-call-quality-dashboard.md:491]:
```suggestion
CQD v3 includes 
```

File: Teams/turning-on-and-using-call-quality-dashboard.md [hunk 4]
```diff
@@ -484,9 +484,17 @@ To compute call count and user count measures, a distinct countif operation is p
...
 
-CQD v2 and CQD v3 have different total counts since CQD v3 has new scenarios not present in CQD v2. Summary Total or Aggregated all-up numbers with no filters are expected to be different.  
+CQDv2 classification for CallSetup failure is inly considered for "Audio" modality only, in CQDv3 this classification happens for every modality (Audio, Video and Appsharing) and is represented in the respective modality stream. 
 
-If the usage scenario includes Skype for Business Server 2019 calls, CQD v3 data includes Skype Bot calls (auto attendant, CVI, Virtual Desktop Interface), Live Events, and PSTN calls. CQD v2 does not use this data. (CQD v3 requires Skype for Business Server 2019 with cloud data connector configured.)
+For Teams, CQDv2 applies the same user feedback to all modalities CQDv3 applies feedback base on modality for Teams.
+
+CQD V3 includes 
+1. Skype for Business Server 2019 calls,
```
Comment (JohanFreelancer9) [inline Teams/turning-on-and-using-call-quality-dashboard.md:492]:
```suggestion
1. Skype for Business Server 2019 calls.
```

File: Teams/turning-on-and-using-call-quality-dashboard.md [hunk 5]
```diff
@@ -484,9 +484,17 @@ To compute call count and user count measures, a distinct countif operation is p
...
 
-CQD v2 and CQD v3 have different total counts since CQD v3 has new scenarios not present in CQD v2. Summary Total or Aggregated all-up numbers with no filters are expected to be different.  
+CQDv2 classification for CallSetup failure is inly considered for "Audio" modality only, in CQDv3 this classification happens for every modality (Audio, Video and Appsharing) and is represented in the respective modality stream. 
 
-If the usage scenario includes Skype for Business Server 2019 calls, CQD v3 data includes Skype Bot calls (auto attendant, CVI, Virtual Desktop Interface), Live Events, and PSTN calls. CQD v2 does not use this data. (CQD v3 requires Skype for Business Server 2019 with cloud data connector configured.)
+For Teams, CQDv2 applies the same user feedback to all modalities CQDv3 applies feedback base on modality for Teams.
+
+CQD V3 includes 
+1. Skype for Business Server 2019 calls, 
+2. Skype Bot calls, such as:auto attendant, call queue, conference announcement service,
```
Comment (JohanFreelancer9) [inline Teams/turning-on-and-using-call-quality-dashboard.md:493]:
```suggestion
2. Skype Bot calls, such as auto attendant, call queue, and conference announcement service.
```

File: Teams/turning-on-and-using-call-quality-dashboard.md [hunk 6]
```diff
@@ -484,9 +484,17 @@ To compute call count and user count measures, a distinct countif operation is p
...
-CQD v2 and CQD v3 have different total counts since CQD v3 has new scenarios not present in CQD v2. Summary Total or Aggregated all-up numbers with no filters are expected to be different.  
+CQDv2 classification for CallSetup failure is inly considered for "Audio" modality only, in CQDv3 this classification happens for every modality (Audio, Video and Appsharing) and is represented in the respective modality stream. 
 
-If the usage scenario includes Skype for Business Server 2019 calls, CQD v3 data includes Skype Bot calls (auto attendant, CVI, Virtual Desktop Interface), Live Events, and PSTN calls. CQD v2 does not use this data. (CQD v3 requires Skype for Business Server 2019 with cloud data connector configured.)
+For Teams, CQDv2 applies the same user feedback to all modalities CQDv3 applies feedback base on modality for Teams.
+
+CQD V3 includes 
+1. Skype for Business Server 2019 calls, 
+2. Skype Bot calls, such as:auto attendant, call queue, conference announcement service, 
+3. Virtual Desktop Interface,
```
Comment (JohanFreelancer9) [inline Teams/turning-on-and-using-call-quality-dashboard.md:494]:
```suggestion
3. Virtual Desktop Interface.
```

File: Teams/turning-on-and-using-call-quality-dashboard.md [hunk 7]
```diff
@@ -484,9 +484,17 @@ To compute call count and user count measures, a distinct countif operation is p
...
+4. Conference Video Interop,
```
Comment (JohanFreelancer9) [inline Teams/turning-on-and-using-call-quality-dashboard.md:495]:
```suggestion
4. Conference Video Interop.
```

**Pull request merged**